deploy/TokenGatewayInit.sol

```diff
@@ -27,7 +27,8 @@ interface L1TokenGatewayLike {
     function l1Router() external view returns (address);
     function inbox() external view returns (address);
     function escrow() external view returns (address);
-    function registerToken(address l1Token, address l2Token) external;
+    function file(bytes32, address) external;
```

Comment:
`escrow()` can be removed now.
This is also relevant for `op-token-bridge`.

Reply:
Fixed, waiting for approval and I'll merge.

Reply:
Fixed on `op-token-bridge` as well: https://github.com/makerdao/op-token-bridge/pull/1/commits/a01b8725f20896390e63a6f65b109c25f8fb823c